scripts/dockerhub_prune.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+# Copyright (c) The Libra Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+######################################################################################################################
+# Takes a slug org/repo ( libra/client ) and deletes all tags with release-* over 90 days and all other              #
+# over 2 days (assumed to be test images).                                                                           #
```

Comment:
We probably don't want to remove all old images do we? Just thinking about the case where you want to grab an image that is a year old for bisecting or something. Or have I misunderstood what this  does?

Reply:
There's no danger in removing old images, all the code to rebuild them are in the libra repo.   @mgorven was fine with a 90 day callback (last 3 releases if we release month, but we seem to be releasing every 2 weeks).    We can adjust the timing when we have a good cadence.

I doubt we'll be able to bisect with docker images more than a year old ( the terraform scripts will have moved on).  In short are systems are too immature to be able to bisect.   Unless your planning something with bors.

Reply:
Pretty sure a couple months old images would be a stretch today....